### PR TITLE
ND-4668 Cover modules with tests

### PR DESCRIPTION
**Pull request by dvovna**

### CODE REVIEW

File: src/modules/array.test.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    test('should return an error if undefined is passed', () => {
+      expect(() => UNIQUE(null as any))
+        .toThrow('Cannot read properties of null (reading \'length\')');
+    });
+
+    test('should throw an error if array length is too big', () => {
+      const testArray = [1, 2, 1, 4];
+
+      expect(() => UNIQUE(testArray))
+        .toThrow(new JexlFunctionExecutionError(`Items size exceeded. Provided ${testArray.length}, maximum 3`));
```
Comment (alexey-maslovsky) [inline src/modules/array.test.ts:38]: we shouldn't test exception message, we just need to check if an exception is instance of the specific error class

File: src/modules/logical.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ export default createModule(({ validateArrayMaxSize }, options) => {
     validateArrayMaxSize(args);
 
     const passedCount = args.reduce((previousPassedCount: number, arg) => {
-      return !!arg ? previousPassedCount + 1 : previousPassedCount;
+      return arg ? previousPassedCount + 1 : previousPassedCount;
```
Comment (alexey-maslovsky) [inline src/modules/logical.ts:35]: why did we update the condition , isn't it a braking change ?

File: src/modules/array.test.ts (hunk 1 above)
Comment (dvovna) [reply src/modules/array.test.ts:38]: wow, why?

Comment (dvovna) [reply src/modules/array.test.ts:38]: discussed, decided to leave as is

**Review by alexey-maslovsky — APPROVED** [review]

**Pull request merged**